### ISSUE TITLE
NullPointer for LongEntry

### ISSUE THREAD

**Issue by vedant1811** [fdoyle/EasyDatastore#6]
I have a interface as:

```
public interface PreferencesStore {
    @Preference("AccessToken")
    ObjectEntry<ServerToken> accessToken();

    @Preference("RefreshTokenAt")
    LongEntry refreshTokenAt();

    @Preference("FollowedTopics")
    ObjectEntry<StoredTopics> topicList();
}
```

the other 2 work fine but I get a  `java.lang.NullPointerException: Attempt to invoke virtual method 'void com.lacronicus.easydatastorelib.LongEntry.put(long)' on a null object reference` when I call `preferencesStore.refreshTokenAt().put(refreshAt);` and also on `MyApplication.instance().getPreferencesStore().refreshTokenAt().get(Long.MAX_VALUE)`

**Comment by fdoyle** [comment]
Hey, good catch. Fixed in 1.0.3, which just went up.

**Comment by vedant1811** [comment]
@fdoyle thanks for the prompt fix :+1:
